constants/appLinks.ts

```diff
@@ -92,6 +100,7 @@ export {
   REQUEST_AN_EVENT_CTA,
   SEMINAR_SERIES_ALL_EPISODES_CTA,
   SEMINAR_SERIES_FULL_ARCHIVE_CTA,
+  TEXTBOOK_DEMO_INTRODUCTION_COURSE_START_LEARNING,
```

Comment:
We use this link in two places but changing the label, please, find a way to reuse the rest instead of creating a new one

Reply:
As discussed offline, I created this related refactoring issue: https://github.com/Qiskit/qiskit.org/issues/1566